[ABCI]: catch up with 0.16.2 / 0.33.4

There seems to be a minor glitch in the integration tests:
```
failures:

---- rpc::abci_info stdout ----
thread 'rpc::abci_info' panicked at 'assertion failed: `(left == right)`
  left: `"0.16.2"`,
 right: `"0.16.1"`', tendermint/tests/integration.rs:37:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    rpc::abci_info

test result: FAILED. 9 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out

error: test failed, to rerun pass '-p tendermint --test integration'
##[error]The process '/usr/share/rust/.cargo/bin/cargo' failed with exit code 101
```

--- a/tendermint/src/rpc/endpoint/abci_info.rs
+++ b/tendermint/src/rpc/endpoint/abci_info.rs
@@ -3,7 +3,6 @@
 use crate::serializers;
 use crate::{block, rpc};
 use serde::{Deserialize, Serialize};
-use serde_bytes;
 
 /// Request ABCI information from a node
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]

--- a/tendermint/tests/integration.rs
+++ b/tendermint/tests/integration.rs
@@ -34,7 +34,7 @@ mod rpc {
     async fn abci_info() {
         let abci_info = localhost_rpc_client().abci_info().await.unwrap();
 
-        assert_eq!(&abci_info.version, "0.16.1");
+        assert_eq!(&abci_info.version, "0.16.2");
         assert_eq!(abci_info.app_version, 1u64);
         // the kvstore app's reply will contain "{\"size\":0}" as data right from the start
         assert_eq!(&abci_info.data, "{\"size\":0}");

